ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRWholeBodyKinematicStreaming.java

```diff
@@ -280,7 +285,10 @@ public void processVRInput()
       vrContext.getController(RobotSide.LEFT).runIfConnected(controller ->
       {
          controller.setAButtonText(streamToController.get() ? "Stop control" : "Start control");
-         controller.setBButtonText(showGhosts.get() ? "Hide ghosts" : "Show ghosts");
+         if (demonstrationMode.get())
+            controller.setBButtonText("Task %d: x%d".formatted(demonstrationTaskIndex, demonstrationCounts[demonstrationTaskIndex]));
```

Comment:
Add a visual demonstration count, which is sort of essential so the operator can track how many they've done.